syntax = "proto3";

package auth;


message AuthRequest {
  string token = 1;
}

message AuthResponse {
  string id = 1;
  string email = 2;
}

service AuthService {
  rpc GetUser (AuthRequest) returns (AuthResponse);
}



message CheckUserRequest {
  string id = 1;
}

message CheckUserResponse {
  bool exists = 1;
}


service CheckUserService {
  rpc CheckUser (CheckUserRequest) returns (CheckUserResponse);
}

